ZAPD/ZRoom/Commands/SetTransitionActorList.cpp

```diff
@@ -50,16 +54,12 @@ string SetTransitionActorList::GenerateSourceCodePass1(string roomName, int base
 	{
 		string actorStr = "";
 
-		if (entry->actorNum < sizeof(ActorList) / sizeof(ActorList[0]))
-			actorStr = ActorList[entry->actorNum];
+		if (entry->actorNum < sizeof(Globals::Instance->cfg->actorList) / sizeof(Globals::Instance->cfg->actorList[0]))
+			actorStr = Globals::Instance->cfg->actorList[entry->actorNum];
```

Comment:
`.at()` has boundaries checks.
```suggestion
			actorStr = Globals::Instance->cfg->actorList.at(entry->actorNum);
```